ntbs-ui-tests/Features/TransferNotification.feature

```diff
@@ -33,7 +33,8 @@ Feature: Transfer notification
     Given I choose to log in with a different account
     Given I have logged in as LeedsServiceUser
     When I navigate to the url of the current notification
-    And I take action on the alert with title Transfer request
+    Then I can see the value Birmingham & Solihull for element with id 'banner-tb-service'
```

Comment:
This is ok, but I wonder if instead it would make sense to change the `When I navigate to the url of the current notification step` to include a generic check that the page has loaded. The trouble with this is that it's quite easy for other people to end up introducing the same bug by accident in the future.

Reply:
What do you think?

Reply:
I think that makes more sense and does future proof the step. I've added in a check for the notification banner title.